assets/js/translator.js

```diff
@@ -20,7 +20,8 @@ if(modeEnabled('translation')) {
 
         function getDstLangs(srcLang) {
             var targets = [];
-            var targetsSeen = {srcLang: true};
+            var targetsSeen = {};
+            targetsSeen[srcLang] = true;
```

Comment:
Seems to work differently: the old one adds a key called 'srcLang', while the new one actually expands the variable.